src/main/java/yahav/subwaysurfer/SubwayStations.java

```diff
@@ -1,33 +1,69 @@
 package yahav.subwaysurfer;
 
+import java.awt.geom.Point2D;
 import java.util.List;
 
 public class SubwayStations {
     List<Station> features;
 
-    public static class Station {
+    public class Station {
+        public List<Station> connections;
         Properties properties;
         Geometry geometry;
-    }
-
-    static class Properties {
-        String objectid;
-        String name;
 
-        public String getObjectID() {
-            return objectid;
+        public Station(List<Station> connections) {
+            this.connections = connections;
         }
 
-        public String getName() {
-            return name;
+        public class Properties {
+            int objectid;
+            String name;
+
+            public int getObjectID() {
+                return objectid;
+            }
+
+            public String getName() {
+                return name;
+            }
         }
-    }
 
-    static class Geometry {
-        List<Double> coordinates;
+        public class Geometry {
+            List<Double> coordinates;
 
-        public List getCoordinates() {
-            return coordinates;
+            public List getCoordinates() {
+                return coordinates;
+            }
+        }
+        public Station findStartStation(Double startLatitude, Double startLongitude) {
+            double closestStartDistance = Double.MAX_VALUE;
+            SubwayStations.Station closestStartStation = SubwayStations.this.features.get(0);
+            Point2D.Double startPoint = new Point2D.Double(startLatitude, startLongitude);
+            for(SubwayStations.Station station : features){
+                Point2D.Double stationStartPoint = new Point2D.Double(station.geometry.coordinates.get(0), station.geometry.coordinates.get(1));
+                double distance = Math.sqrt((stationStartPoint.getX()-startPoint.getX()*2) + (stationStartPoint.getY() - startPoint.getY()*2));
+                if(closestStartDistance > distance){
+                    closestStartDistance = distance;
+                    closestStartStation = station;
+                }
+            }
+            return closestStartStation;
+        }
+        public Station findEndStation(Double endLatitude, Double endLongitude){
```

Comment:
This method does the exact same thing as the previous method. You only need one method that does this.